src/app/pages/swap/components/swap-assets-pair/swap-assets-pair.tsx

```diff
@@ -21,14 +21,22 @@ export function SwapAssetsPair() {
     return null;
   }
 
+  const formattedSwapAmountBase = formatMoneyWithoutSymbol(
+    createMoneyFromDecimal(
+      Number(swapAmountBase),
+      swapAssetBase.name,
+      swapAssetBase.balance.decimals
+    )
+  );
+
   return (
     <SwapAssetsPairLayout
       swapAssetBase={
         <SwapAssetItemLayout
           caption="You will swap"
           icon={swapAssetBase.icon}
           symbol={swapAssetBase.name}
-          value={swapAmountBase}
+          value={formattedSwapAmountBase}
```

Comment:
```suggestion
          value={formatMoneyWithoutSymbol(swapAmountBase)}
```

small suggestion, but good idea to format at last minute